avaframe/ana3AIMEC/dfa2Aimec.py

```diff
@@ -17,59 +17,25 @@
 log = logging.getLogger(__name__)
 
 
-def writeAimecPathsFile(cfgSetup, avaDir, dirName='com1DFA'):
-    """ Write a pathFile to inform Aimec where its input data is located """
-
-    # Initialise DEM
-    inputDir = os.path.join(avaDir, 'Inputs')
-    dem = glob.glob(inputDir+os.sep+'*.asc')
-
-    # Load parameters for Aimec postprocessing
-    pressureLimit = float(cfgSetup['pressureLimit'])
-    domainWidth = float(cfgSetup['domainWidth'])
-
-    # Path to com1DFA output in Aimec format
-    workDir = os.path.join(avaDir, 'Work', 'ana3AIMEC', dirName)
-
-    # Create empty variable
-    emptyVar = ""
-
-    with open(os.path.join(workDir, 'aimecPathFile.txt'), 'w') as pfile:
-        pfile.write('pathPressure=%s,\n' % (os.path.join(workDir, 'dfa_pressure')))
-        pfile.write('pathMass=%s,\n' % (os.path.join(workDir, 'dfa_mass_balance')))
-        pfile.write('pathDocDamage=%s,\n' % (emptyVar))
-        pfile.write('pathDocRadar=%s,\n' % (emptyVar))
-        pfile.write('pathNumInfo=%s,\n' % (emptyVar))
-        pfile.write('pathFlowHeight=%s,\n' % (os.path.join(workDir, 'dfa_depth')))
-        pfile.write('pathAvalanchePath=%s,\n' % (os.path.join(inputDir, 'avalanche_path.xyz')))
-        pfile.write('calcPressureLimit=%s,\n' % (pressureLimit))
-        pfile.write('domainWidth=%s,\n' % (domainWidth))
-        pfile.write('pathDepoArea=%s,\n' % (emptyVar))
-        pfile.write('pathAOI=%s,\n' % (emptyVar))
-        pfile.write('pathDHM=%s,\n' % (dem[0]))
-        pfile.write('pathEnergy=%s,\n' % (emptyVar))
-        pfile.write('pathVelocity=%s,\n' % (emptyVar))
-        pfile.write('pathResult=%s,\n' % (os.path.join(workDir, 'AimecResults')))
-
-
-def extractMBInfo(avaDir, simName=''):
+def extractMBInfo(avaDir, pathDict, simNameInput=''):
```

Comment:
Maybe make it more clear that these functions target com1DFA. Either we rename the file to com1DFA2Aimec.py or the functions to something like extractCom1DFAMbInfo...

Reply:
made an issue

Reply:
Done